docs/docs/template-tags-filters.md

```diff
@@ -57,6 +57,20 @@ It's possible to specify the source of the attribute value by writing it as a ke
 <input type="text" id="first_name" name="first_name" />
 ```
 
+It's possible to specify the default value of an attribute by declaring it with the [`var`](#var) tag
+
+```twig
+{# image component #}
+{% var loading="lazy" %}
```

Comment:
Hey @jlopinto, I don't think it's a default value per se, what's happening is that you're "redefining" the `loading` with the value of `"lazy"`.

If you try something different that `"lazy"` the loading will still be set to `"lazy"`:
```html
{# Usage #}
{% image src="..." alt="..." loading="eager" %}

{# Output #}
<img src="..." alt="..." loading="lazy" />
```

But would be really cool if we could pass default values when defining the `{% attrs %}`, something like:
```html
<img {% attrs src loading="lazy" alt %}>
```

Which unfortunately doesn't work, it's kind of a short version of yours. A nice alternative for that is to use the `|default` template filter to achieve that:
```suggestion
{% var loading=loading|default:"lazy" %}
```